cognite/neat/core/workflow/base.py

```diff
@@ -244,9 +245,18 @@ def method_runner():
                 if self.task_builder:
                     sync_str = step.params.get("sync", "false")
                     sync = sync_str.lower() == "true" or sync_str == "1"
-                    new_flow_message = self.task_builder.start_workflow_task(
+                    start_status = self.task_builder.start_workflow_task(
                         workflow_name=step.params.get("workflow_name", ""), sync=sync, flow_message=self.flow_message
                     )
+                    if start_status.is_success and start_status.workflow_instance.state == WorkflowState.COMPLETED:
+                        new_flow_message = start_status.workflow_instance.flow_message
+                    else:
+                        logging.error(f"Workflow step {step.id} failed to start workflow task")
+                        if start_status.is_success:
+                            raise Exception(start_status.workflow_instance.last_error)
+                        else:
+                            raise Exception(start_status.status_text)
```

Comment:
I think we should have a custom exception here and avoid the generic Exception class.

Reply:
Yes